lib/qiita/markdown/filters/external_link.rb

```diff
@@ -0,0 +1,35 @@
+module Qiita
+  module Markdown
+    module Filters
+      class ExternalLink < HTML::Pipeline::Filter
+        def call
+          return doc unless hostname
```

Comment:
@yujinakayama Hmm, I don't expect that the `hostname` context parameter is required, but the `hostname` parameter can be optional, I think. If the `ExternalLink` filter is applied and the `hostname` is NOT specified, this `ExternalLink` filter does nothing, I expected. Therefore, I wrote this line to return right away if the `hostname` is not passed. How do you think this behaviour?